### PR TITLE
Swerve SysID and minor auto fixes.

### PR DESCRIPTION
**Pull request by chaoticthegreat**

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -358,14 +361,16 @@ private void configureIntake() {
         .whileTrue(
             intake.setVoltage(
                 -IntakeConstants.kIntakeIntakeVoltage, -IntakeConstants.kPassthroughIntakeVoltage));
-    driver.rightTrigger().whileTrue(intake.intakeIn());
```
Comment (realjoshuau) [inline src/main/java/frc/robot/RobotContainer.java:361]: why was this removed?

Comment (chaoticthegreat) [reply src/main/java/frc/robot/RobotContainer.java:361]: conflicted with the rotation triggers

Comment (realjoshuau) [reply src/main/java/frc/robot/RobotContainer.java:361]: 👍
